content/en/docs/concepts/architecture/leases.md

```diff
@@ -38,3 +38,43 @@ rest of the system. While not particularly useful on its own, this provides a me
 discover how many instances of `kube-apiserver` are operating the Kubernetes control plane.
 Existence of kube-apiserver leases enables future capabilities that may require coordination between
 each kube-apiserver.
+
+You can inspect Leases owned by each kube-apiserver by checking for lease objects in the `kube-system` namespace
+with the name `kube-apiserver-<sha256-hash>`. Alternatively you can use the label selector `k8s.io/component=kube-apiserver`:
+
+```shell
+$ kubectl -n kube-system get lease -l k8s.io/component=kube-apiserver
+NAME                                        HOLDER                                                                           AGE
```

Comment:
Please split this - see [don't include the command prompt](https://kubernetes.io/docs/contribute/style/style-guide/#don-t-include-the-command-prompt) and [separate commands from output](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output).